apps/coscrad-frontend/src/store/slices/resources/shared/connected-resources/connected-resources-panel.tsx

```diff
@@ -11,12 +11,15 @@ export interface ConnectedResourcesPanelProps {
 export const ConnectedResourcesPanel = ({
     compositeIdentifier,
 }: ConnectedResourcesPanelProps): JSX.Element => {
-    const loadable = useLoadableConnectionsToResource(compositeIdentifier);
+    const {
+        isLoading,
+        errorInfo,
+        data: connections,
+    } = useLoadableConnectionsToResource(compositeIdentifier);
 
-    const Presenter = displayLoadableWithErrorsAndLoading(
-        ConnectedResourcesPanelPresenter,
-        wrapArrayProps
-    );
+    if (errorInfo) return <ErrorDisplay {...errorInfo} />;
 
-    return <Presenter {...loadable} />;
+    if (isLoading || connections === null) return <Loading />;
```

Comment:
We really should be wrapping the behaviour in with the `displayLoadable...` helper.

Reply:
done